src/usdb_syncer/song_loader.py

```diff
@@ -745,15 +745,28 @@ def _write_headers(ctx: _Context) -> None:
 
     if path := ctx.out.audio.path(ctx.locations, temp=True):
         _set_audio_headers(ctx, version, path)
+    else:
+        ctx.txt.headers.mp3 = None
```

Comment:
I'd find it more comprehensible if audio headers were only set in a single place. So e.g. `_set_audio_headers` should also handle the `None` case. It also doesn't need to be passed `path`, which is derived from `ctx`.